# Enumerated measurement types:
uint8 t_undefined=0
uint8 t_measurementCalculated=1
uint8 t_measurementInferred=2
uint8 t_measurementActual=3

bool absoluteMeasurement

uint8 msgType
float32 distance
float32 angle
